be/src/storage/lake/general_tablet_writer.cpp

```diff
@@ -90,6 +90,9 @@ Status HorizontalGeneralTabletWriter::flush_segment_writer(SegmentPB* segment) {
         uint64_t footer_position = 0;
         RETURN_IF_ERROR(_seg_writer->finalize(&segment_size, &index_size, &footer_position));
         _data_size += segment_size;
+        auto segment_file = _seg_writer->segment_path();
+        auto pos = segment_file.find_last_of("/");
+        _files_to_size.emplace(segment_file.substr(pos + 1), segment_size);
```

Comment:
Is it the segment size after compression or the size before compression?

Reply:
It's the size of physical segment file